### PR TITLE
Add login screen to v3 sample

### PR DESCRIPTION
**Pull request by cardoso**
Preview
image

#no_changelog

### CODE REVIEW

File: StreamChat_v3.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1526,7 +1550,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = D48F367GRM;
 				INFOPLIST_FILE = Sample_v3/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```
Comment (VojtaStavik) [inline StreamChat_v3.xcodeproj/project.pbxproj:1553]: Can we leave it to 11 and use @available checks for the iOS13+ parts? We use this app for testing the older versions, too.

File: Sample/Cocoapods/Podfile.lock [hunk 2]
```diff
@@ -0,0 +1,67 @@
+PODS:
```
Comment (VojtaStavik) [inline Sample/Cocoapods/Podfile.lock:1]: This shouldn't be here. It's the v2 Sample app. The v3 Sample doesn't need any dependencies so far.

File: StreamChat_v3.xcodeproj/project.pbxproj (hunk 1 above)
Comment (cardoso) [reply StreamChat_v3.xcodeproj/project.pbxproj:1553]: @VojtaStavik yes. I'll just need to remove Combine.

Comment (VojtaStavik) [reply StreamChat_v3.xcodeproj/project.pbxproj:1553]: It's ok to link against it (it's weakly linked), just put any usage of it behind the availability check.

**Pull request merged**